pkgs/applications/misc/blender/derivation.nix

```diff
@@ -0,0 +1,280 @@
+{ config, stdenv, lib, fetchFromGitea, fetchzip, boost, cmake, ffmpeg, gettext, glew
+, ilmbase, libepoxy, libXi, libX11, libXext, libXrender
+, libjpeg, libpng, libsamplerate, libsndfile
+, libtiff, libwebp, libGLU, libGL, openal, opencolorio, openexr, openimagedenoise, openimageio, openjpeg
+, openvdb, libXxf86vm, tbb, alembic
+, zlib, zstd, fftw, opensubdiv, freetype, jemalloc, ocl-icd, addOpenGLRunpath
+, jackaudioSupport, libjack2
+, cudaSupport, cudaPackages ? { }
+, hipSupport, rocmPackages # comes with a significantly larger closure size
+, colladaSupport, opencollada
+, spaceNavSupport, libspnav
+, makeWrapper
+, pugixml, llvmPackages, SDL, Cocoa, CoreGraphics, ForceFeedback, OpenAL, OpenGL
+, waylandSupport, pkg-config, wayland, wayland-protocols, libffi, libdecor, libxkbcommon, dbus
+, potrace
+, openxr-loader
+, embree, gmp, libharu
+, openpgl
+, mesa
+, runCommand
+, version, hashes, isLTS, pythonPackages, withPackages, xvfb-run
+}:
+
+let
+  python = pythonPackages.python;
+  optix = fetchzip {
+    # url taken from the archlinux blender PKGBUILD
```

Comment:
Link?

Reply:
This was part of the original file. I can still find it if it's necessary.

Reply:
Arch also has updated to Optix 7.6. I don't understand the differences between the versions, but Blender build instructions currently states to use 7.3, so I'm leaving it as 7.3.